generated/nifake/session.py

```diff
@@ -600,7 +600,7 @@ def __init__(self, resource_name, options={}, id_query=False, reset_device=False
 
 
         Returns:
-            session (nifake.Session): A session object representing the device.
+            session (int): A session object representing the device.
```

Comment:
This got reverted, appears to be wrong now.

Reply:
Yet another place that was still using `'python_type'`. Fixed.